CHANGELOG.md

```diff
@@ -1,16 +1,28 @@
 # Change Log
 
+## v8.0.1 (February 9, 2024)
+
+### Fixed
+- Chopped header for the landscape orientation ([#531](https://github.com/etn-ccis/blui-react-native-component-library/issues/531)).
+- Icon alignment in `<DrawerNavItem>` component.
```

Comment:
can you please change to log to "Icon alignment in `<InfoListItem>` component."
as this is the component causing issues